rust/headless-client/src/main.rs

```diff
@@ -151,7 +151,23 @@ enum Cmd {
 const VERSION: &str = env!("CARGO_PKG_VERSION");
 const RELEASE: &str = concat!("headless-client@", env!("CARGO_PKG_VERSION"));
 
-fn main() -> Result<()> {
+#[expect(
+    clippy::print_stderr,
+    reason = "No logger is active when we are printing this error."
+)]
+fn main() {
+    match try_main() {
+        Ok(()) => {}
+        Err(e) => {
+            // Print chain of errors manually to avoid it looking like a crash with stacktrace.
+            eprintln!("{e:#}");
```

Comment:
out of curiosity, why not use `tracing`? you're worried about errors in tracing init or it is it an aesthetic preference?